Doc/library/time.rst

```diff
@@ -43,16 +43,13 @@ An explanation of some terminology and conventions is in order.
 
 .. index::
    single: Year 2000
+   single: 2-digit years
    single: Y2K
```

Comment:
I think you should remove these old index entries, too.

Reply:
I wasn't sure if they'd still be useful as redirection, but agree they're likely not needed and will remove.